torchtext/data/example.py

```diff
@@ -11,7 +13,29 @@ class Example(object):
 
     @classmethod
     def fromJSON(cls, data, fields):
-        return cls.fromdict(json.loads(data), fields)
+        # Modified by JahoJiang in 2019.07.17
```

Comment:
We don't leave personal information here. People see your contribution from the commit history :).

Reply:
Oops, I forgot to delete it when I submitted it, sorry. 

I will delete the information and fix unit test.